src/hashset.c

```diff
@@ -1291,23 +1291,25 @@ size_t hashsetScan(hashset *t, size_t cursor, hashsetScanFunction fn, void *priv
             /* Advance cursor. */
             cursor = nextCursor(cursor, mask);
         } else {
-            /* Let table0 be the the smaller table and table1 the bigger one. */
-            int table0, table1;
-            if (t->bucketExp[0] <= t->bucketExp[1]) {
-                table0 = 0;
-                table1 = 1;
+            int growing = t->bucketExp[0] <= t->bucketExp[1];
+            int table_small, table_big;
+            if (growing) {
+                table_small = 0;
+                table_big = 1;
             } else {
-                table0 = 1;
-                table1 = 0;
+                table_small = 1;
+                table_big = 0;
             }
 
-            size_t mask0 = expToMask(t->bucketExp[table0]);
-            size_t mask1 = expToMask(t->bucketExp[table1]);
+            size_t mask_small = expToMask(t->bucketExp[table_small]);
+            size_t mask_big = expToMask(t->bucketExp[table_big]);
```

Comment:
If we use the 5-letter word "large" instead of "big", it aligns better with "small". :-)

```suggestion
            size_t mask_small = expToMask(t->bucketExp[table_small]);
            size_t mask_large = expToMask(t->bucketExp[table_large]);
```